src/panels/logbook/ha-logbook.ts

```diff
@@ -67,6 +84,10 @@ class HaLogbook extends LitElement {
     }
     const previous = this.entries[index - 1];
     const state = item.entity_id ? this.hass.states[item.entity_id] : undefined;
+    const item_username =
+      item.context_user_id && item.context_user_id in this._userid_to_name
+        ? this._userid_to_name[item.context_user_id]
+        : undefined;
```

Comment:
This is not needed, if the key doesn't exist the value would be undefined anyway